openapi/custom_objects_spec.json

```diff
@@ -435,6 +435,298 @@
       }
     }
   },
+  "/apis/{group}/{version}/{plural}/{name}/status": {
+    "parameters": [
+      {
+        "name": "group",
+        "in": "path",
+        "required": true,
+        "description": "the custom resource's group",
+        "type": "string"
+      },
+      {
+        "name": "version",
+        "in": "path",
+        "required": true,
+        "description": "the custom resource's version",
+        "type": "string"
+      },
+      {
+        "name": "plural",
+        "in": "path",
+        "required": true,
+        "description": "the custom resource's plural name. For TPRs this would be lowercase plural kind.",
+        "type": "string"
+      },
+      {
+        "name": "name",
+        "in": "path",
+        "required": true,
+        "description": "the custom object's name",
+        "type": "string"
+      }
+    ],
+    "get": {
+      "description": "read status of the specified cluster scoped custom object",
+      "consumes": [
+        "*/*"
+      ],
+      "produces": [
+        "application/json",
+        "application/yaml",
+        "application/vnd.kubernetes.protobuf"
```

Comment:
Custom resources don't support protobuf. I'm not sure if yaml is supported though. 

After a discussion with @yliaog, we want to keep the changes between beta release and GA release minimum for python-client release (I'll update devel documentation about the release procedure). We decided to not include this new API change in python-client v7.0.0. Let's fix this spec and introduce the API to v8.